### ISSUE TITLE
Xvideos and XNXX low quality only

### ISSUE THREAD

**Issue by DrDRECoder** [ytdl-org/youtube-dl#24781]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.03.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
XVIDEOS
youtube-dl -F -v https://www.xvideos.com/video55065063/15347003/0/reckless_in_miami_-_autumn_falls_-_just_a_dip
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-F', u'-v', u'https://www.xvideos.com/video55065063/15347003/0/reckless_in_miami_-_autumn_falls_-_just_a_dip']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.03.24
[debug] Python version 2.7.13 (CPython) - Linux-4.9.0-11-amd64-x86_64-with-debian-9.12
[debug] exe versions: none
[debug] Proxy map: {}
[XVideos] 55065063: Downloading webpage
[info] Available formats for 55065063:
format code  extension  resolution note
mp4-low      mp4        unknown


XNXX
youtube-dl -F -v https://www.xnxx.com/video-m3i0t02/stepmom_christie_stevens_sneak_into_stepsons_bed_and_fucks_him_beside_sleeping_girlfriend_
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-F', u'-v', u'https://www.xnxx.com/video-m3i0t02/stepmom_christie_stevens_sneak_into_stepsons_bed_and_fucks_him_beside_sleeping_girlfriend_']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.03.24
[debug] Python version 2.7.13 (CPython) - Linux-4.9.0-11-amd64-x86_64-with-debian-9.12
[debug] exe versions: none
[debug] Proxy map: {}
[XNXX] m3i0t02: Downloading webpage
[info] Available formats for m3i0t02:
format code  extension  resolution note
low          mp4        unknown
```

## Description

Xvideos had 2 links (mp4-low and mp4-high). Now mp4-low is available only.
Same problem with XNXX (low only).

**Comment by Oneboy1979** [comment]
Hi, i can download hires video withoud problems:
 youtube-dl -F https://www.xnxx.com/video-m3i0t02/stepmom_christie_stevens_sneak_into_stepsons_bed_and_fucks_him_beside_sleeping_girlfriend_
[XNXX] m3i0t02: Downloading webpage
[XNXX] m3i0t02: Downloading m3u8 information
[info] Available formats for m3i0t02:
format code  extension  resolution note
low          mp4        unknown
high         mp4        unknown
hls-250p     mp4        444x250     155k
hls-360p     mp4        640x360     423k
hls-480p     mp4        854x480     763k
hls-720p     mp4        1280x720   1327k  (best)

 youtube-dl -F https://www.xvideos.com/video55065063/15347003/0/reckless_in_miami_-_autumn_falls_-_just_a_dip
[XVideos] 55065063: Downloading webpage
[XVideos] 55065063: Downloading m3u8 information
[info] Available formats for 55065063:
format code  extension  resolution note
mp4-low      mp4        unknown
mp4-high     mp4        unknown
hls-250p     mp4        444x250     155k
hls-360p     mp4        640x360     423k
hls-480p     mp4        854x480     763k
hls-720p     mp4        1280x720   1327k
hls-1080p    mp4        1920x1080  2760k  (best)

regards Oneboy1979

**Comment by DrDRECoder** [comment]
I think my IP was restricted by xvideos. It works with proxy.
